lib/task/jobs/jobWorkerTask.class.php

```diff
@@ -223,6 +267,11 @@ protected function activateTerminationHandlers()
         // Define shutdown function
         register_shutdown_function(function () {
             $this->log('Job worker stopped.');
+
+            if (arMemprofUtils::getMemprofEnabled()) {
+                $memprof_filename = arMemprofUtils::createMemprofGrindFile();
+                $this->log(sprintf('Memprof enabled. Dumping shutdown grind file: %s', $memprof_filename));
```

Comment:
Nitpick: you could call arMemprofUtils::createMemprofGrindFile() directly here, instead of creating a single use variable.